app/install/kubernetes.md

```diff
@@ -10,9 +10,15 @@ links:
   kubectl: "https://cloud.google.com/container-engine/docs/quickstart#install_the_gcloud_command-line_interface"
   gh_tag: "https://github.com/Kong/kong-dist-kubernetes/tree/1.0.0"
   minikube: "https://github.com/Kong/kong-dist-kubernetes/blob/master/minikube/README.md"
+  kubeapps_hub: "https://hub.kubeapps.com/charts/stable/kong"
+  gh_repo: "https://github.com/Kong/kong-dist-kubernetes/""
```

Comment:
the trailing quote on this line breaks the header.

Reply:
Thanks @darrenjennings for review, I pushed fix.